packages/demo/usecase-demo/src/modules/DisclaimerModal.tsx

```diff
@@ -34,26 +33,31 @@ export function DisclaimerModal() {
     setStorageDevMode(isDevMode);
   }, [isDevMode]);
 
-  useEffect(() => {
-    if (open == false) {
-      setStorageDiscaimerViewed(true);
+  function onModalOpenChange(open: boolean) {
+    setOpen(open);
+    if (!open) {
+      setStorageDisclaimerViewed(true);
     }
-  }, [open]);
+  }
+
+  if (isStorageDisclaimerViewed) {
+    return false;
+  }
 
-  if (!isStorageDiscaimerViewed) {
-    return (
-      <Dialog open={open} onOpenChange={setOpen}>
-        <DialogContent>
-          <DialogHeader>
+  return (
+    <Dialog open={open} onOpenChange={onModalOpenChange}>
+      <DialogContent className="pt-6">
+        <DialogDescription className="pb-6">
```

Comment:
Maybe just a `<div>` here as `DialogDescription` becomes a `p` tag, hence it should not contain divs.